docs/how-to-build-with-ddn/with-stripe.mdx

```diff
@@ -0,0 +1,243 @@
+---
+sidebar_position: 8
+sidebar_label: With Stripe
+description: "Learn the basics of Hasura DDN and how to get started with a Stripe account."
+keywords:
+  - hasura ddn
+  - graphql api
+  - getting started
+  - guide
+  - stripe
+---
+
+import Prereqs from "@site/docs/_prereqs.mdx";
+
+# Get Started with Hasura DDN and Stripe
+
+## Overview
+
+This tutorial takes about ten minutes to complete. You'll learn how to:
+
+- Set up a new Hasura DDN project
+- Connect it to a Stripe account
+- Generate Hasura metadata
+- Create a build
+- Run your first query
+
+Additionally, we'll familiarize you with the steps and workflows necessary to iterate on your API.
+
+This tutorial assumes you're starting with an existing [Stripe account](https://stripe.com/) which has some
+[products](https://docs.stripe.com/api/products) and [checkout sessions](https://docs.stripe.com/api/checkout/sessions).
+Though, by the end, you'll be able to query any information stored in Stripe.
+
+Hasura will never modify your source schema.
+
+<Prereqs />
+
+## Tutorial
+
+### Step 1. Authenticate your CLI
+
+```sh title="Before you can create a new Hasura DDN project, you need to authenticate your CLI:"
+ddn auth login
+```
+
+This will launch a browser window prompting you to log in or sign up for Hasura DDN. After you log in, the CLI will
+acknowledge your login, giving you access to Hasura Cloud resources.
+
+### Step 2. Scaffold out a new local project
+
+```sh title="Next, create a new local project:"
+ddn supergraph init my-project && cd my-project
+```
+
+Once you move into this directory, you'll see your project scaffolded out for you. You can view the structure by either
+running `ls` in your terminal, or by opening the directory in your preferred editor.
+
+### Step 3. Initialize your Stripe connector
+
+```sh title="In your project directory, run:"
+ddn connector init my_stripe -i
+```
+
+From the dropdown, select `hasura/stripe` (you can type to filter the list), then hit enter to accept the default of all
+the options.
+
+You'll be prompted for an environment variable:
+
+| Variable                   | Description                                                                                                    | Example                               |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------- | ------------------------------------- |
+| `STRIPE_BEARER_AUTH_TOKEN` | An auth token retrievable from your [API Keys dashboard in Stripe](https://dashboard.stripe.com/test/apikeys). | `sk_<test\|prod>_<unique_identifier>` |
```

Comment:
let's say here explicitly whether to use the stripe publishable key or secret key